src/test/java/bio/terra/service/auth/iam/sam/SamIamTest.java

```diff
@@ -87,7 +87,7 @@ class SamIamTest {
   private static final String GROUP_NAME = "firecloud_group_name";
   private static final String ADMIN_EMAIL = "samAdminGroupEmail@a.com";
   private final SamConfiguration samConfig =
-      new SamConfiguration("https://sam.dsde-dev.broadinstitute.org", ADMIN_EMAIL, 10, 30, 60);
+      new SamConfiguration("https://sam.dsde-dev.broadinstitute.org", ADMIN_EMAIL, 0, 0, 0);
```

Comment:
Drive by test speedup 😄 
These configuration values set the retry wait for the Sam calls. One of the tests was waiting for the retry to fail, so by setting these to 0, we're able to speed up the tests from 35 seconds down to 5 seconds.